test/internet/test-dns-setservers-type-check.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+require('../common');
+const { addresses } = require('../common/internet');
+const assert = require('assert');
+const dns = require('dns');
+const resolver = new dns.promises.Resolver();
+const dnsPromises = dns.promises;
+const promiseResolver = new dns.promises.Resolver();
```

Comment:
I don't think this needs to be in `test/internet`, right? It only sets servers, not actually tries to send out DNS requests?

Reply:
@addaleax Thanks for your review.I will move this test file to `test/parallel`. Is it OK?